src/timing_analysis/plot_utils.py

```diff
@@ -2084,17 +2091,25 @@ def plot_residuals_orb(fitter, restype = 'postfit', plotsig = False, avg = False
             r_b_label = ""
         else:
             r_b_label = rcvr_bcknds[inds][0]
-        # Get plot preferences
-        if 'fmt' in kwargs.keys():
-            mkr = kwargs['fmt']
-        else:
+    # Get plot preferences
+    if 'fmt' in kwargs.keys():
```

Comment:
It looks like possible indentation changes between the old file and the new. Is that intended?

~ Joe G.